configure.ac

```diff
@@ -131,11 +131,22 @@ if test "x$AGENTS_LIST" = x; then
 fi
 
 if test "x$AGENTS_LIST" != xall; then
+	TMP=""
 	for j in $AGENTS_LIST; do
 		if ! test -d fence/agents/$j; then
 			AC_ERROR([Agent $j does not exists])
 		fi
+
+		if test "$j" = "lib" -o "$j" = "nss_wrapper" -o "$j" = "autodetect"; then
+			continue
+		fi
+
+		AGENT=`find $srcdir/fence/agents/$j -name '*.py' -printf '%P'`
```

Comment:
It would have to be a combination of updates to configure.ac and Makefiles to be able to be able to select from all the agents due to some special logic for some of the agents (e.g. the ones with symlinks and the ones that are in C).